samples/system-test/acl.test.js

```diff
@@ -54,49 +52,47 @@ after(async () => {
   }
 });
 
-it('should print acl for a bucket', async () => {
-  const out = await exec(`${cmd} print-bucket-acl ${bucketName}`);
+it('should print acl for a bucket', () => {
+  const out = execSync(`${cmd} print-bucket-acl ${bucketName}`);
   assert.match(out, /OWNER: project-editors-/);
   assert.match(out, /OWNER: project-owners-/);
   assert.match(out, /READER: project-viewers-/);
 });
 
-it('should print acl for a file', async () => {
-  const out = await exec(`${cmd} print-file-acl ${bucketName} ${fileName}`);
+it('should print acl for a file', () => {
+  const out = execSync(`${cmd} print-file-acl ${bucketName} ${fileName}`);
```

Comment:
Is the problem with the integration tests potentially that `execSync` is returning a `Buffer` and not a String?

Reply:
This was addressed above!  Good callout.